static/js/components/GroupSources.jsx

```diff
@@ -0,0 +1,351 @@
+import React, { useEffect, Suspense } from "react";
+import PropTypes from "prop-types";
+import { useSelector, useDispatch } from "react-redux";
+
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import Grid from "@material-ui/core/Grid";
+import Chip from "@material-ui/core/Chip";
+import Link from "@material-ui/core/Link";
+import PictureAsPdfIcon from "@material-ui/icons/PictureAsPdf";
+
+import MUIDataTable from "mui-datatables";
+import { makeStyles } from "@material-ui/core/styles";
+
+import Tooltip from "@material-ui/core/Tooltip";
+import GroupIcon from "@material-ui/icons/Group";
+
+import dayjs from "dayjs";
+
+import { ra_to_hours, dec_to_hours } from "../units";
+import * as SourcesAction from "../ducks/sources";
+import styles from "./CommentList.css";
+import ThumbnailList from "./ThumbnailList";
+import UserAvatar from "./UserAvatar";
+import ShowClassification from "./ShowClassification";
+
+const VegaPlot = React.lazy(() => import("./VegaPlot"));
+
+const useStyles = makeStyles((theme) => ({
+  chip: {
+    margin: theme.spacing(0.5),
+  },
+  source: {},
+}));
+
+const GroupSources = ({ route }) => {
+  const dispatch = useDispatch();
+  const sources = useSelector((state) => state.sources.groupSources);
+  const groups = useSelector((state) => state.groups.user);
+  const { taxonomyList } = useSelector((state) => state.taxonomies);
+  const classes = useStyles();
+
+  // Color styling
+  const userColorTheme = useSelector(
+    (state) => state.profile.preferences.theme
+  );
+
+  const commentStyle =
+    userColorTheme === "dark" ? styles.commentDark : styles.comment;
+
+  // Load the group sources
+  useEffect(() => {
+    dispatch(SourcesAction.fetchGroupSources({ group_id: route.id }));
```

Comment:
```suggestion
    dispatch(SourcesAction.fetchGroupSources({ group_ids: [route.id] }));
```

Reply:
Ok but why?

Reply:
Because group_ids should always be a list. see my comments/suggestions elsewhere

Reply:
I agree. But I think in this case it doesn't matter because the URLSearchParams turns it into a string that then gets parsed on the back-end. Otherwise, I can't understand how it worked for me until now...

Reply:
get_query_argument converts it to a list
If you were just passing a string like "2" before, it was likely only working because you were iterating over each character of the string, and each character happened to be a valid group ID.

Reply:
Yes, that was the problem I had with the test, but I think that if I send a string to URLSearchParams and then call toString() on that I will get a string with comma-separated values. And I am parsing for that on the back-end. That's why I don't think it matters much... But I certainly agree it looks more clear.